common/stats/status_server.cpp

```diff
@@ -92,6 +98,12 @@ StatusServer::StatusServer(uint16_t port, EndPointToOPMap op_map)
       ret += itor->second(nullptr);
     }
 
+    for (const auto& endpoint_name : extra_stats_endpoints_) {
```

Comment:
Tested and looked at the ostrich-fetcher code. Applications should not provide top keys like "metric" or "counters", they will overwrite the old one in the original "/stats.txt" result. I added comments for that.